src/components/shared/ButtonWithPanel.css

```diff
@@ -8,43 +8,3 @@
   flex-flow: column nowrap;
   align-items: stretch;
 }
-
-/**
- * This wrapper uses overflow: hidden to clip the actual button.
- * This clip is needed for the vertical slide animation that the
- * profile sharing composite button does.
- * The clip is on .buttonWithPanelButtonWrapper and not on
- * .buttonWithPanel because the latter would clip the panel. */
```

Comment:
We don't use this animation anymore so I removed the wrapper and this CSS.

Reply:
Thanks for the cleanup!